posnums.c

```diff
@@ -0,0 +1,70 @@
+/* Jared Wasserman -- posnums.c */
+/*This program takes two numbers from the users and prints the average, weather each number is prime, and wheteher the two numbers are relatively prime.*/
```

Comment:
Nice job overall, but you didn't print out whether each number was prime in some cases, so you logic is a bit off. Add more comments throughout code so I can let you know where you went wrong. You also generated two warnings, which you shouldn't do.